docs/runbooks/VirtOperatorRESTErrorsBurst.md

```diff
@@ -2,11 +2,9 @@
 
 ## Meaning
 
-For the last 10 minutes or longer, over 80% of the REST calls made to
-`virt-operator` pods have failed.
-
-This usually indicates that the `virt-operator` pods cannot connect to the API
-server.
+This alert fires when more than 80% of the REST calls in the `virt-operator`
+pods failed in the last 5 minutes. This usually indicates that the
+`virt-operator` pods cannot connect to the API server.
```

Comment:
Same comment as on the `VirtApiRESTErrorsBurst.md` file

Reply:
Same :)

Reply:
https://github.com/kubevirt/monitoring/pull/240#discussion_r1590437383